### DISCUSSION TITLE
Faust.JS + WooCommerce

### DISCUSSION THREAD

**Discussion by frzsombor** [wpengine/faustjs#1864] [Q&A]
Has anyone tried creating a WooCommerce webshop with Faust.JS? I'm looking for a general headless WP solution that I can use for basic blog sites, sites with custom content (custom post types) and even WooCommerce. I know there are a few projects already supporting headless/static WooCommerce, I just want to know if Faust supports it or not. Has anyone tried it or is it at least theoretically possible, but would require huge amount of research and work?

**Comment by AttaM1rza** [comment]
any update ?
